### DISCUSSION TITLE
autoComplete and mySQL

### DISCUSSION THREAD

**Discussion by kohein613** [TarekRaafat/autoComplete.js#343] [Q&A]
Good evening, I have been playing around with autoComplete, and while I see several general examples, I have found nothing explaining how to connect to and retrieve the auto complete items from a (mySQL) database.

**Comment by folknor** [comment]
That's because it's outside the scope of autoComplete.js. What you want is a normal JS MySQL tutorial, and to learn JS in general, and then plug them together.

**Reply by folknor** [reply]
Trying to think of an analogy that can be easily understood, and the best one I came up with in 1 minute is:

What you're doing is the same as asking a random driver of a random car how the road they're on is produced and applied. Expecting the driver to know is just not rational. It's not their job.

**Comment by kohein613** [comment]
Thanks!

After I posted, I went to shower, and was pondering the issue and thinking
along those lines...

Menashe

**Reply by MhmedRjb** [reply]
so do you find a way?

**Reply by kohein613** [reply]
I moved to Selectize very shortly after posting my question. It works nicely.

**Comment by khiemlamNI** [comment]
This issue is under review for closure due to its age. If it remains relevant, it will be tracked in the issue listed below. Please provide any updates there. If no updates are received, the issue will be closed next Wednesday at 5 PM Austin time.

Issue link: https://github.com/ni/ni-adas-replay-hil-plugins/issues/75
